### PR TITLE
L3 milestone defined

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

**Review by mjzac — CHANGES_REQUESTED** [review]
We can ask them to add logic to retrieve overdue, due later and due today items. Here is the previous WD201 milestone for the same level. https://www.pupilfirst.school/targets/1725

**Comment by avishekjana** [comment]
@mjzac I've updated the milestone task, please review once.

**Comment by avishekjana** [comment]
@mjzac @vikispeaks I've updated the milestone based on review. Please check once.

**Comment by avishekjana** [comment]
@mjzac @vikispeaks I've updated the milestone based on review. Please check once.

File: nodejs-deep-dive/milestone/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+console.log("My Todo-list\n\n")
+
+console.log("Overdue\n")
+var overdues = todos.overdue()
+var formattedOverdues = todos.toDisplayableList(overdues)
+console.log(formattedOverdues)
+console.log("\n\n")
+
+console.log("Due Today\n")
+var todaydues = todos.dueToday()
```
Comment (mjzac) [inline nodejs-deep-dive/milestone/README.md:100]:
```suggestion
let itemsDueToday = todos.dueToday()
```

File: nodejs-deep-dive/milestone/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+var formattedTodaydues = todos.toDisplayableList(todaydues)
```
Comment (mjzac) [inline nodejs-deep-dive/milestone/README.md:101]:
```suggestion
let formattedItemsDueToday = todos.toDisplayableList(itemsDueToday)
```

File: nodejs-deep-dive/milestone/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+console.log(formattedTodaydues)
+console.log("\n\n")
+
+console.log("Due Later\n")
+var laterdues = todos.dueLater()
```
Comment (mjzac) [inline nodejs-deep-dive/milestone/README.md:106]:
```suggestion
let itemsDueLater = todos.dueLater()
```

**Pull request merged**